guaranteed-message-delivery/request_handler.bal

```diff
@@ -0,0 +1,41 @@
+import ballerina/http;
+import ballerinax/rabbitmq;
+
+type Patient record {|
```

Comment:
Let's move common types to a separate file and generate them first. Will help when writing the content too IMO, since the producer will be written before the consumer?

Reply:
+1